runtime/bcutil/ClassFileOracle.hpp

```diff
@@ -924,6 +924,10 @@ class NameAndTypeIterator
 	bool annotationRefersDoubleSlotEntry() const { return _annotationRefersDoubleSlotEntry; }
 	bool isInnerClass() const { return _isInnerClass; }
 
+	bool needsStaticConstantInit() const {
```

Comment:
Should probably be formatted like the rest of the getters.

Reply:
I've pushed a new commit with adjusted formatting